src/backend/browser/sandbox.ts

```diff
@@ -33,6 +33,23 @@ import {
 // TODO: base pathを設定できるようにするか、ビルド時埋め込みにする
 const toStaticPath = (fileName: string) => `/${fileName}`;
 
+function onReceivedIPCMsg<T extends IpcRendererOn>(listeners: T): void;
+function onReceivedIPCMsg(listeners: {
+  [key: string]: (event: unknown, ...args: unknown[]) => unknown;
+}) {
+  window.addEventListener(
```

Comment:
ここどうやら使われてないっぽいので、メモ書きコメントだけお願いできると 🙇 
```suggestion
  // NOTE: もしブラウザ本体からレンダラへのメッセージを実装するならこんな感じ
  window.addEventListener(
```

@yamachu ちなみに例示するのが実装時の意図だった、という認識で合ってそうでしょうか 👀 
[このあたり](https://github.com/VOICEVOX/voicevox/pull/1345/files#diff-b50f6753c347be6ca10c4e787a6d6d9217570fb679430c5ef9652d9dc6b0b651R184-R193)っぽそう。
すんごい前のことを聞いちゃってすみません 🙇 

---

このあたり解説すると、electronではバック→フロントへの通信が必要で（エンジンプロセスがエラー吐いたとか）、その通信用なんですよね。
でもブラウザ版ではいまのとこ必要なさそう。
たぶんこれから先もいらなそう･･･？だとしたら消しても良さそう？

だけどそれは流石にこのPRの範疇を超えてそう！
なのでちょっとメモコメントをお願いしてみた次第です！

Reply:
resolvedになってるけど一応当時の意図を…

当初WorkerにファイルReadの処理を投げて、それを受け取るためにipcの処理を先んじて実装しておこうという意図で置いていました。
実装するとしたらこうなりますだけ書いておいたやつですね…

なので

> でもブラウザ版ではいまのとこ必要なさそう。
> たぶんこれから先もいらなそう･･･？だとしたら消しても良さそう？

消しても良さそうではあります！

Reply:
おおなるほどです！！
自信が持てました、ありがとうございます！！